src/main/java/org/wso2/carbon/esb/connector/auth/AmazonLambdaAuthConnector.java

```diff
@@ -0,0 +1,438 @@
+            String requestPayload = "";
+            final Map<String, String> payloadParametersMap = ParameterNamesMap.getPayloadParameterNamesMap();
+            final StringBuilder payloadBuilder = new StringBuilder();
+            for (Map.Entry<String, String> entry : payloadParametersMap.entrySet()) {
+                String key = entry.getKey();
+                String tempParam = payloadParametersValueMap.get(key);
+                if (StringUtils.isNotEmpty(tempParam)) {
+                    payloadParamsMap.put(payloadParametersMap.get(key),
+                            tempParam.replaceAll
+                                    (AmazonLambdaConstants.TRIM_SPACE_REGEX, AmazonLambdaConstants.EMPTY_STR));
+                }
+            }
+            final SortedSet<String> payloadKeys = new TreeSet<>(payloadParamsMap.keySet());
+            if (payloadKeys.equals(AmazonLambdaConstants.PAYLOAD)) {
+                payloadBuilder.append(payloadParamsMap.get(AmazonLambdaConstants.PAYLOAD));
+            } else {
+                for (String key : payloadKeys) {
+                    String payloadValues = payloadParamsMap.get(key);
+                    payloadBuilder.append('"');
+                    payloadBuilder.append(key);
+                    payloadBuilder.append('"');
+                    payloadBuilder.append(':');
+                    /**
+                     * Checks for "{" and "[", to omit putting payload value inside quotation mark, which
+                     * represents either the value has nested-payload-like parameter or is an array. In both
+                     * case the value need not to be inside the quotation mark.
+                     */
+                    if (payloadValues.substring(0, 1).equals("{") || payloadValues.substring(0, 1).equals("[")) {
+                        payloadBuilder.append(payloadValues)
+                                .append(',');
+                    }
+                    /**
+                     * Checks for the parameter which contains integer value. If it is such parameters then
+                     * it directly put the value without putting inside quotation mark while building the payload.
+                     */
+                    else if (key.equals(AmazonLambdaConstants.API_PUBLISH) ||
+                            key.equals(AmazonLambdaConstants.API_TIMEOUT)) {
+                        payloadBuilder.append(payloadValues)
+                                .append(',');
+                    }
+                    /**
+                     *If both conditions mentioned above fails then the values should be inside the quotation mark.
+                     * This condition does so.
+                     */
+                    else {
+                        payloadBuilder.append('"');
+                        payloadBuilder.append(payloadValues);
+                        payloadBuilder.append('"');
```

Comment:
you can append like 
payloadBuilder.append('"').append(payloadValues).append('"');

Reply:
Fix with 408c7bd